components/style/style_adjuster.rs

```diff
@@ -28,6 +28,12 @@ impl<'a, 'b: 'a> StyleAdjuster<'a, 'b> {
         StyleAdjuster { style }
     }
 
+    /// Whether we should skip display item-based display fixups.
+    fn skip_item_based_display_fixup(&self) -> bool {
+        // Pseudos never get their display fixed up, except ::before or ::after.
+        self.style.pseudo.as_ref().map_or(false, |p| !p.is_before_or_after())
```

Comment:
Can we reuse `p.skip_item_based_display_fixup()` for now, or remove that method as well as the corresponding constant at the same time?

Reply:
Sure, will do.

Reply:
Done.